src/test/java/org/opensearch/security/setting/DeprecatedSettingsTest.java

```diff
@@ -58,6 +58,6 @@ public void testCheckForDeprecatedSettingFoundLegacy() {
 
         checkForDeprecatedSetting(settings, "legacyKey", "properKey");
 
-        verify(logger).deprecate(eq("legacyKey"), anyString(), any());
+        verify(logger).deprecate(eq("legacyKey"), anyString(), any(), any());
```

Comment:
should we wait for #2428 to be backported for this change to go through. I don't mind either way

Reply:
I think we can merge without waiting for backport for this small change